### PR TITLE
Delete the Huntable trait for walls and gates.

### LINKED ISSUE #11760: Enemies try to hunt down walls in Survival01
image

This means the mission won't properly end when the player should loose.

### PR DESCRIPTION
**Pull request by MunWolf**
This closes #11760 by making the walls non huntable, we might want to make some other things non huntable, like trees.

### CODE REVIEW

**Comment by WolfGaming** [comment]
To elaborate this is because nearly everything inherits from ^ExistsInWorld which defines Huntable, and I don't think huntable is mentioned anywhere else in the yaml definitions so everything is huntable.

**Comment by Phrohdoh** [comment]
I updated the title (directly from the commit message) for searchability and clarity, hope you don't mind.

**Comment by GraionDilach** [comment]
Well, neither defines `MustBeDestroyed`, so I don't think there is any harm in this PR. :+1:

**Comment by WolfGaming** [comment]
I was wondering if we want to do the -Huntable on trees and other actors just to remove a redundant trait, any opinions?

**Comment by pchote** [comment]
Wouldn't it make sense to remove `Huntable` from ExistsInWorld, and then add it only to the defaults that actually want it? (basically only player-owned stuff)

**Comment by WolfGaming** [comment]
I would think so but I didn't want to do it until I got an ok from the peoples, and if someone wants huntable to be on trees/walls they can always add them back in the map.yaml that needs it.

**Comment by WolfGaming** [comment]
PR updated.

File: mods/ra/rules/defaults.yaml [hunk 1]
```diff
@@ -296,6 +297,7 @@
 		MaxMoveDelay: 750
 
 ^ArmedCivilian:
+	Huntable:
```
Comment (GraionDilach) [inline mods/ra/rules/defaults.yaml]: You don't need this, `^ArmedCivilian` is a template extension of `CivInfantry`, and every user of this template will have the `Huntable` from `Soldier` already.

**Comment by Mailaender** [comment]
:+1: This is less fragile. Compare https://github.com/OpenRA/OpenRA/pull/2855.

**Comment by obrakmann** [comment]
Could I convince you to do the same for the other mods?

**Comment by WolfGaming** [comment]
@obrakmann It is done.

**Comment by obrakmann** [comment]
Thanks. Looks fine, :+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/b08e3a68023f9f8d395dcff7cc5f18a27e2f110a)

**Pull request merged**